content/W2016/W16-event-big-data-panel.md

```diff
@@ -1,7 +1,7 @@
 Title: Big Data Panel
 Date: 2016-02-15
 Category: Events
-Tags: big data, panels
+Tags: panels
```

Comment:
I think we should keep tags like 'big data' and 'graphics'. Makes it easy to search for events imo.